src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/PackageSourceTelemetry.cs

```diff
@@ -461,6 +471,11 @@ internal static class Duration
                     internal const string Header = "http.duration.header";
                 }
             }
+
+            internal static class PackageNamespaces
+            {
+                internal const string AreNamespacesEnabled = "PackageNamespaces.AreNamespacesEnabled";
```

Comment:
It affects this. 
Please change the name of the telemetry we emit to `PackageSourceMapping.IsMappingEnabled`
cc @kartheekp-ms

Reply:
@nkolev92 
Ok. I changed the name, please check now.